tests/app/piping/test_plumber.py

```diff
@@ -23,74 +26,10 @@ def test_plumb_item(self):
         item = ObjectFromDict({
             "id": "item1",
             "templatable_properties": ['description'],
-            "description": "Property One is {simple.property_one}, while Property Two is {simple.property_two}"
+            "description": "Property One is {{simple.property_one}}, while Property Two is {{simple.property_two}}"
         })
 
-        self.assertEquals(item.description, "Property One is {simple.property_one}, while Property Two is {simple.property_two}")
-
-        self.plumber.plumb_item(item)
+        with patch('app.piping.plumber.render_template_string', return_value="Property One is value one, while Property Two is value two"):
+            self.plumber.plumb_item(item)
 
         self.assertEquals(item.description, "Property One is value one, while Property Two is value two")
-
-        item2 = ObjectFromDict({
-            "id": "item2",
-            "templatable_properties": ['property_one', 'property_two', 'property_three'],
-            "property_one": "Date is {dates.first_april_2016:%-d %B %Y}",
```

Comment:
I have added a unit test for the function and we already have integration tests covering this